server/graphql/invoice/src/mutations/outbound_return/update.rs

```diff
@@ -0,0 +1,121 @@
+use async_graphql::*;
+
+use graphql_core::{
+    standard_graphql_error::{validate_auth, StandardGraphqlError},
+    ContextExt,
+};
+use graphql_types::types::InvoiceNode;
+use service::invoice::outbound_return::update::{
+    UpdateOutboundReturn as ServiceInput, UpdateOutboundReturnError as ServiceError,
+};
+use service::{
+    auth::{Resource, ResourceAccessRequest},
+    invoice::outbound_return::update::UpdateOutboundReturnStatus,
+};
+
+use super::insert::OutboundReturnLineInput;
+
+#[derive(InputObject)]
+#[graphql(name = "UpdateOutboundReturnInput")]
+pub struct UpdateInput {
+    pub id: String,
+    // supplier_id: String,
+    status: Option<UpdateOutboundReturnStatusInput>,
+    outbound_return_lines: Vec<OutboundReturnLineInput>,
+}
+
+#[derive(Enum, Copy, Clone, PartialEq, Eq, Debug)]
+pub enum UpdateOutboundReturnStatusInput {
+    Allocated,
```

Comment:
Copied the same statuses as outbound shipment, but is Allocated a relevant status for an outbound return? You're returning specific stock, so the idea of a placeholder line which you then allocate seems irrelevant right?